### PR TITLE
Feature - Add CarrierWave

### PR DESCRIPTION
**Pull request by kamilsdz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ if you use HTTPS
 ```
 cd motohelper/
 gem install bundler
+brew install imagemagick (on OSX)
```
Comment (fab-ian) [inline README.md]: Add this info please:
sudo apt-get install ruby-mini-magick (on  Ubuntu)

File: app/dashboards/user_dashboard.rb [hunk 2]
```diff
@@ -65,7 +72,7 @@ class UserDashboard < Administrate::BaseDashboard
   FORM_ATTRIBUTES = %i[
     name
     email
-    password
```
Comment (fab-ian) [inline app/dashboards/user_dashboard.rb]: This field is required to add an AdminUser from Administrate. Please don't remove it.

Comment (kamilsdz) [reply app/dashboards/user_dashboard.rb]: Do some tests. If this field is added, admin can't change any field (for example: avatar or email). I recommend to create AdminUser by changing Regular User to Admin User (We must add field "type") - it's the simplest way.

File: README.md (hunk 1 above)
Comment (kamilsdz) [reply README.md]: Thanks for tip!

File: app/dashboards/user_dashboard.rb (hunk 2 above)
Comment (fab-ian) [reply app/dashboards/user_dashboard.rb]: OK, we will discuss about this topic on Hangout.
Now leave the password removed.

**Pull request merged**